package.json

```diff
@@ -10,17 +10,17 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/graywolf336/temporary-rocketlets-ts-definition.git"
+    "url": "git+https://github.com/graywolf336/temporary-apps-ts-definition.git"
   },
   "keywords": [
-    "rocketlets"
+    "apps"
   ],
   "author": "Bradley Hilton <bradley.hilton@rocket.chat>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/graywolf336/temporary-rocketlets-ts-definition/issues"
+    "url": "https://github.com/graywolf336/temporary-apps-ts-definition/issues"
```

Comment:
:top:  ok i'll stop :grin:

Reply:
The repository wasn't transferred when this commit was done ;)